src/PaymentSuite/PaymillBundle/Resources/views/Paymill/scripts.html.twig

```diff
@@ -51,28 +51,25 @@
         oPaymill.bOkExpiry = paymill.validateExpiry(oPaymill.sExpiryMonth, oPaymill.sExpiryYear);
         oPaymill.bOkCvv = paymill.validateCvc(oPaymill.nCvv);
 
-
-
         if (oPaymill.bOkNumber !== true) {
             var oLiNumber = document.createElement('li');
-            oLiNumber.innerText = 'Card number invalid';
+            oLiNumber.innerHTML = 'Card number invalid';
```

Comment:
my bad: it appears ``innerText`` is not a standard, this may explain why it's not printed (http://stackoverflow.com/a/19032002)

I suggest the use of ``value`` property :)